### DISCUSSION TITLE
I am not able to see icons in Document card although I am using class components.

### DISCUSSION THREAD

**Discussion by Nileshkhonde5233** [microsoft/fluentui#30474] [Q&A]
Icons Which I am Using Are Of Fluent ui's and I am Following IconsProps For the css.
Example-        const closeIcon: IIconProps = { iconName: 'ChromeClose', styles: { root: { color: 'white', padding: '18px 0px', fontSize: "15px" } } };

**Comment by khmakoto** [comment]
Hi @Nileshkhonde5233, are you by any chance not initializing the icons before trying to use them?

**Reply by Nileshkhonde5233** [reply]
yes I am using initializeIcons() as i have already use fluent ui but the catch is cards are getting deprecated and we are switching in document card but the example all they have given on doc are about the functional componennts and i have doubt that they support the icons in class component or not

**Reply by Nileshkhonde5233** [reply]
Why Card is deprectaed and what is the best alternative for it can you please suggest
